src/test/regress/expected/typed_streams.out

```diff
@@ -4,11 +4,10 @@ SET search_path TO test_typed_streams,public;
 CREATE STREAM s0 (x integer, y integer);
 DROP TABLE s0;
 ERROR:  "s0" is not a table
-HINT:  Use DROP STREAM to remove a stream.
+HINT:  Use DROP FOREIGN TABLE to remove a foreign table.
```

Comment:
Yea, it kind of sucks but that's all we can do without having a stream relkind:

https://github.com/pipelinedb/pipelinedb/pull/1235/files#diff-2b1247515e9aec2cf5fc7b716eb677c7L256